forage-android/src/main/java/com/joinforage/forage/android/pos/ForageTerminalSDK.kt

```diff
@@ -708,14 +712,10 @@ class ForageTerminalSDK internal constructor(private val posTerminalId: String)
      *   }
      * }
      * ```
-     *
      * @param params The [PosRefundPaymentParams] parameters required for refunding a Payment.
      * @return A [ForageAPIResponse][com.joinforage.forage.android.network.model.ForageApiResponse]
      * indicating the success or failure of the
-     * PIN capture. On success, returns `Nothing`.
-     * On failure, the response includes a list of
-     * [ForageError][com.joinforage.forage.android.network.model.ForageError] objects that you can
-     * unpack to troubleshoot the issue.
+     * PIN capture.
```

Comment:
nit: This was present before, but can we change (in this/future PR) all instances of PIN capture to say "secure PIN submission" or something alike

Given that capture is used in the context of "payment capture"